src/common/meta/Cargo.toml

```diff
@@ -59,7 +59,7 @@ session.workspace = true
 snafu.workspace = true
 store-api.workspace = true
 strum.workspace = true
-table.workspace = true
+table = { workspace = true, features = ["testing"] }
```

Comment:
Should we use `optional = true` here and import `table` in `[dev-dependencies]`?